build.gradle

```diff
@@ -0,0 +1,90 @@
+plugins {
+    id 'com.palantir.docker' version '0.25.0'
+    id 'org.jetbrains.kotlin.jvm' version "${kotlin_version}"
+    id 'application'
+}
+
+apply plugin: 'kotlin-kapt'
+
+group = 'com.exactpro.th2'
+version = release_version
+
+sourceCompatibility = 11
+targetCompatibility = 11
+
+compileKotlin {
+    kotlinOptions.jvmTarget = '11'
+}
+
+compileTestKotlin {
+    kotlinOptions.jvmTarget = '11'
+}
+
+repositories {
+    mavenCentral()
+
+    maven {
+        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+    }
+
+    maven {
+        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+    }
+
+    mavenLocal()
+
+    configurations.all {
+        resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+        resolutionStrategy.cacheDynamicVersionsFor 0, 'seconds'
+    }
+}
+
+jar {
+    manifest {
+        attributes(
+                'Created-By': "${System.getProperty('java.version')} (${System.getProperty('java.vendor')})",
+                'Specification-Title': '',
+                'Specification-Vendor': 'Exactpro Systems LLC',
+                'Implementation-Title': project.archivesBaseName,
+                'Implementation-Vendor': 'Exactpro Systems LLC',
+                'Implementation-Vendor-Id': 'com.exactpro',
+                'Implementation-Version': project.version
+        )
+    }
+}
+
+dependencies {
+    api platform('com.exactpro.th2:bom:3.2.0')
+
+    implementation 'com.exactpro.th2:conn-dirty-tcp-core:2.0.0'
+    implementation 'com.exactpro.th2:common:3.40.0-TH2-3789-2817159492-SNAPSHOT'
```

Comment:
Could we use release version instead

Reply:
This is a dev build with session-group support. Did we release session-groups?